tests/util.go

```diff
@@ -123,12 +146,18 @@ func WaitUntilResourceReadyByLabelTestNamespace(resourceType, label, query, expe
 
 func CreateUser(username string) {
 	By(fmt.Sprintf("Wait until user %s is created ", username))
-	execute(Result{cmd: "oc", verb: "create", resourceType: "user", resourceName: username})
+	_, stderr, err := ktests.RunCommandWithNS("", "oc", "create", "user", username)
+	if !strings.Contains(stderr, fmt.Sprintf("Error from server (AlreadyExists): users.user.openshift.io \"%s\" already exists", username)) {
+		ktests.PanicOnError(err)
+	}
 }
 
 func DeleteUser(username string) {
 	By(fmt.Sprintf("Wait until user %s is deleted", username))
-	execute(Result{cmd: "oc", verb: "delete", resourceType: "user", resourceName: username})
+	_, stderr, err := ktests.RunCommandWithNS("", "oc", "delete", "user", username)
+	if !strings.Contains(stderr, fmt.Sprintf("Error from server (NotFound): users.user.openshift.io \"%s\" not found", username)) {
+		ktests.PanicOnError(err)
```

Comment:
We do not want to interrupt the whole test run here.

Reply:
I have removed:
```suggestion
		ktests.PanicOnError(err)
```